src/GSoft.Extensions.MediatR.Analyzers/AnalyzerReleases.Shipped.md

```diff
@@ -0,0 +1,18 @@
+## Release 0.1.0
```

Comment:
The `Microsoft.CodeAnalysis.CSharp.Workspaces` packages comes with an analyzer similar to the one for public API tracking / breaking changes.

The analyzers requires this file to be created and kept up-to-date. It also helps detect if we make a mistake (rename a rule ID for example)